### DISCUSSION TITLE
Is this a 3-platform library? Web, Android, and iOS?

### DISCUSSION THREAD

**Discussion by xml** [joinflux/capacitor-segment#24] [Q&A]
Hi! Thanks for this plugin! 

I wouldn't have asked about 3-platform support, as it's not mentioned in README... except there's a conspicuous web.ts file, and it seems to attach methods to the window object. Also, the README doesn't provide any examples of toggling between one version of Segment library for web, and another for mobile. So... is it intended to support all three uses, rather than needing to toggle?

**Comment by anonimitoraf** [comment]
Hey @xml, I believe we used it for our web app around 1.5 years ago. We're now only on mobile. Did you get to try it out and found that it doesn't work?
